Umbraco-StarterKit/Creating-Content/How-to-Set-Up-Landingpage/index.md

```diff
@@ -0,0 +1,138 @@
+---
+versionFrom: 8.0.0
+---
+
+# How to set up a landingpage
+
+This is a guide designed to help you make a nice landing page, for your website.
+In each step, one feature will be covered. It is important to note that this guide will not cover all options you will have in terms of features.
+
+The Result should look somewhat like this:
+
+![What the page will somewhat look like](images/Frontend-landingpage.gif)
+
+## First Step
+
+- Start by right-clicking the Umbraco Hotel node
+- Choose Create and select page
+- Name your landing page and you are ready to start creating your new landing page
+- In the content section of your page, navigate to the widget section and click "Add content"
+
+## Hero
+
+We will start by adding a Greeting to our page, this will be done using the hero widget, as it allows for having text on top of a background image in a good way.
```

Comment:
```suggestion
We will start by adding a greeting to our page. This will be done using the Hero widget, as it allows for having text on top of a background image in a good way.
```